IBPSA/Fluid/Storage/Examples/StratifiedUnloadAtMinimumTemperature.mo

```diff
@@ -144,18 +145,20 @@ equation
           {48,112}},color={255,0,255}));
   connect(yTop.y, valTop.y) annotation (Line(points={{101,120},{122,120},{122,-8}},
                                color={0,0,127}));
-  connect(yMid.y, valMed.y) annotation (Line(points={{101,80},{142,80},{142,-48}},
+  connect(yMid.y,valMid. y) annotation (Line(points={{101,80},{142,80},{142,-48}},
```

Comment:
Whitespace should be added after the comma, not after the dot.

Reply:
This is fixed in 6ee8e0d.